disconnected_registry/docker-registry-sync.py

```diff
@@ -52,6 +55,12 @@
 # Set the default release version incase unspecified
 release_version = '3.7'
 
+# Set registry.redhat.io auth as default
+remote_registry_auth_url = 'https://registry.redhat.io/auth/realms/rhcc/protocol/redhat-docker-v2/auth?service=docker-registry'
+
+# List of auth-required registry
+auth_required_registry = ['registry.redhat.io', 'quay.io']
```

Comment:
Recommend removing hardcoded restriction on specific registries

Reply:
Agree. Managing this will be tedious.